fossunited/www/payment-failed.html

```diff
@@ -0,0 +1,120 @@
+{% extends "templates/foss_base.html" %} {% block title %} {{ _("Your Payment
+Was failedful!") }}{% endblock %} {% block page_content %}
+<div id="payment-failed-page">
+  <div class="payment-failed">
+    <div id="failed-icon-div">
+      <img class="failure-icon container-center icon" src="/assets/fossunited/icons/failure-icon.svg">
+          </img>
+    </div>
+    <div class="text-center">
+      <p>
+        Unforunately, your payment failed :(
+        <br />
+        Please contact the admins or drop us a email at <b>tickets@fossunited.org</b>.
+        <br />
+        <br />
+      </p>
+    </div>
+  </div>
+</div>
+
+<script>
+  setTimeout(function() {
+    window.location.href = "https://fossunited.org"
+  }, 5000);
+</script>
+<style>
+  #payment-failed-page {
+    width: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    padding: 10px 10px;
```

Comment:
Let's try and use rems instead of px? That practice, using rems instead of px, rems are more relative and responsive.
```suggestion
    padding: 10px 10px;
```